src/plugins/kibana/server/routes/api/ingest/register_simulate.js

```diff
@@ -18,13 +19,16 @@ export function registerSimulate(server) {
       const simulateApiDocument = request.payload;
       const body = ingestSimulateApiKibanaToEsConverter(simulateApiDocument);
 
+      const handleResponse = processESIngestSimulateResponse;
```

Comment:
Didn't realize the module was exporting a default. In that case you can do `import handleResponse  from '../../../lib/process_es_ingest_simulate_response';` which is even better :)